### PR TITLE
fix: iOS 15 photo picker thread

### PR DESCRIPTION
**Pull request by brainbicycle**
The type of this PR is: **fix**

This PR resolves [CX-1794]

### Description

- Moves photo picker configuration to main thread
- Fixes some warnings in the photo picker module and navigation button

Not totally clear if this was causing crash but was causing photo picker not to show up on my device on iOS 15 simulator but wasn't crashing.
@pvinis if you could try running on device and seeing if this fixes I would appreciate 🙏 

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- Fix photo picker presentation on iOS 15 - Brian

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-1794]: https://artsyproduct.atlassian.net/browse/CX-1794

### CODE REVIEW

File: Artsy/Views/Styled_Subclasses/Buttons/ARNavigationButton.m [hunk 1]
```diff
@@ -9,7 +9,7 @@
 @interface ARNavigationButton ()
 
 @property (nonatomic, strong, readonly) UILabel *primaryTitleLabel;
-@property (nonatomic, strong, readonly) UILabel *subtitleLabel;
+@property (nonatomic, strong, readonly) UILabel *secondaryTitleLabel;
```
Comment (brainbicycle) [inline Artsy/Views/Styled_Subclasses/Buttons/ARNavigationButton.m:12]: UIButton has a subtitleLabel property that was conflicting with this

File: emission/Pod/Classes/EigenCommunications/ARPHPhotoPickerModule.m [hunk 2]
```diff
@@ -90,7 +90,7 @@ - (void)picker:(PHPickerViewController *)picker didFinishPicking:(NSArray<PHPick
 
         double delayInSeconds = 3.0;
         dispatch_time_t timeout = dispatch_time(DISPATCH_TIME_NOW, delayInSeconds * NSEC_PER_SEC);
-        intptr_t status = dispatch_group_wait(imageLoadGroup, timeout);
```
Comment (brainbicycle) [inline emission/Pod/Classes/EigenCommunications/ARPHPhotoPickerModule.m:93]: unused

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- iOS user-facing changes (Fix photo picker presentation on iOS 15 - Brian)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against effadadff67c4a89832f13041296a6fdc7f85d70
</p>

**Review by pvinis — APPROVED** [review]
tested and works 👍

**Pull request merged**